spec/index.html

```diff
@@ -7410,6 +7410,110 @@ <h5>TZ</h5>
             </div>
           </section>
         </section>
+
+        <!-- QT -->
+        <section id="func-quoted-triples">
+          <h4>Functions on Quoted Triples</h4>
+          <section id="func-triple">
+            <h5>TRIPLE</h5>
+            <pre class="prototype nohighlight">
+              <span class="return">quoted triple</span>  <span class="operator">TRIPLE</span> (<span class="type RDFterm">RDF term</span> <span class="name">subj</span>, <span class="type RDFterm">RDF term</span> <span class="name">pred</span>, <span class="type RDFterm">RDF term</span> <span
+class="name">obj</span>)
+            </pre>
+            <pre class="query nohighlight">
+                  <span class="operator">&lt;&lt;</span> term1 term2 term3 <span class="operator">&gt;&gt;</span>
+            </pre>
+            <p>
+              If the 3-tuple (<code style="color:black">term1</code>, 
+              <code style="color:black">term2</code>, 
+              <code style="color:black">term3</code>)
+              is an <a data-cite="RDF12-CONCEPTS#dfn-rdf-triple">RDF triple</a>
+              (that is, the subject is a IRI, 
+              <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a> or a 
+              <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank node</a>,
+              and the predicate is an IRI),
+              the function returns a quoted triple with these three elements.
+              Otherwise, the function raises an error.
+            </p>
+            <p>
+              A <code>TRIPLE</code> can also be written with a 
+              <a href="#rExprQuotedTriple">quoted triple expression</a>
```

Comment:
```suggestion
              As a shorthand notation, the <code>TRIPLE</code> function can also be written as a 
              <a href="#rExprQuotedTriple">quoted triple expression</a>
```

Reply:
Good idea. Works well with the syntactic restriction of the form.